src/components/alert/alert.html

```diff
@@ -9,6 +9,12 @@
     <link rel="stylesheet" href="../../css/main.css" />
     <link rel="stylesheet" href="../../css/docs.css" />
     <link rel="stylesheet" href="./alert.css" />
```

Comment:
You can put all of them in a single css file and import that css file.

Reply:
Thanks, I've added common CSS.